### PR TITLE
ISPN-6131 Fix script removal (and other scripting fixes)

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-6131

### CODE REVIEW

**Comment by vblagoje** [comment]
@tristantarrant I'll take a look and integrate!

File: scripting/src/test/java/org/infinispan/scripting/ScriptingTaskManagerTest.java [hunk 1]
```diff
@@ -28,15 +30,19 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
...
-      try (InputStream is = this.getClass().getResourceAsStream("/test.js")) {
-         String script = TestingUtil.loadFileAsString(is);
-         scriptCache.put(SCRIPT_NAME, script);
-      }
    }
 
    public void testTask() throws Exception {
-      String result = (String) taskManager.runTask(SCRIPT_NAME, new TaskContext().addParameter("a", "a")).get();
+      ScriptingManager scriptingManager = cacheManager.getGlobalComponentRegistry().getComponent(ScriptingManager.class);
+      ScriptingTest.loadScript(scriptingManager, TEST_SCRIPT);
```
Comment (danberindei) [inline scripting/src/test/java/org/infinispan/scripting/ScriptingTaskManagerTest.java]: Maybe move this method to a separate `ScriptingTests` class.

**Comment by tristantarrant** [comment]
Moved the utility method to a common class

**Comment by galderz** [comment]
Looks good, testing...

**Comment by galderz** [comment]
Testing fails:

```
Failed tests:
  SecureScriptingTest.testSimpleScript » Test
Expected exception java.lang.Secu...
  SecureScriptingTest.loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95 » StackOverflow
  SecureScriptingTest.loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95->loadScript:95 » StackOverflow
  SecureScriptingTest.testSimpleScriptWithEXECPermissionsWrongRole » Test
Expec...

Tests run: 22, Failures: 4, Errors: 0, Skipped: 1
```

**Comment by galderz** [comment]
Integrated, the tests are being fixed in #4017

**Pull request closed without merging**